src/styles/blocks/calendar.scss

```
@@ -0,0 +1,58 @@
+@import "_variables.scss";
+
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  width: ($calendar-column-side-size * 7 + $calendar-column-gap-width * 6);
+  gap: $calendar-column-gap-width;
+  font-family: Arial, sans-serif;
+  font-size: 30px;
+
+  &__day {
+    width: $calendar-column-side-size;
+    height: $calendar-column-side-size;
+    border: 1px solid #000;
+    background-color: #eee;
+    display: none;
+    justify-content: center;
+    align-items: center;
+    cursor: pointer;
+    transition: all 0.5s ease-in;
```

do not use all property for trasition. You change only transform and background-color
```suggestion
    transition: transform 0. 5s, background-color 0.5s ease-in;
```